library/src/main/res/layout/chucker_list_item_transaction.xml

```diff
@@ -40,27 +41,51 @@
         tools:text="GET /path/to/some/resource?goes=here" />
 
     <TextView
-        android:id="@+id/host"
+        android:id="@+id/graphqlPath"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/chucker_half_grid"
         android:ellipsize="end"
         android:maxLines="2"
+        android:textAppearance="@style/Chucker.TextAppearance.ListItem"
```

Comment:
I'm unsure we want to use this style as it will make it bold. Ideally having it styled as the `host` would be better 👍

Reply:
Done! @cortinico

Reply:
Nope this is good to go 👍